src/binary/ibc-relayer/commands/start.ts

```diff
@@ -81,12 +92,17 @@ export async function start(flags: Flags) {
     mnemonic,
     srcConnection,
     destConnection,
+    // TODO: make configurable
+    runOnce: true,
+    // once per day: 86400s
+    maxAgeA: 86400,
+    maxAgeB: 86400,
   };
 
-  run(options, logger);
+  await run(options, logger);
```

Comment:
no need for `await` here but it doesn't hurt

Reply:
I guess not, but if I neither `await` nor return the Promise, then if the caller tries `await`, it will not wait for the final promise with the heart of the work.

I guess we don't do it that way here, but it is a habit after a few gotchas with unawaited promises